pkgs/by-name/li/liblzf/package.nix

```diff
@@ -0,0 +1,139 @@
+{
+  lib,
+  stdenv,
+  autoreconfHook,
+  fetchDebianPatch,
+  fetchurl,
+  pkg-config,
+  testers,
+  validatePkgConfig,
+  writeText,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "liblzf";
+  version = "3.6";
+  shlib_version = "1:0:0"; # for libtool
+
+  # used for changelog
+  cvsTag = "rel-${builtins.replaceStrings [ "." ] [ "_" ] finalAttrs.version}";
```

Comment:
Can we move this to a let in?

Reply:
Resolved in 05221c935ab2ade0d6c328caed2032bc6d8e09d0 :)